src/status_im2/contexts/chat/home/chat_list_item/view.cljs

```diff
@@ -0,0 +1,124 @@
+(ns status-im2.contexts.chat.home.chat-list-item.view
+  (:require [clojure.string :as string]
+            [utils.re-frame :as rf]
+            [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.core :as quo]
+            [status-im2.contexts.chat.home.chat-list-item.style :as style]
+            [status-im2.common.home.actions.view :as actions]
+
+            ;;TODO move to status-im2
```

Comment:
same here. remove comment now?

Reply:
[status-im.utils.datetime :as time] still needs to be moved